src/UI/PlayerSetupInfo.java

```diff
@@ -17,61 +15,74 @@
  */
 public class PlayerSetupInfo
 {
-  public ArrayList<Integer> coList = new ArrayList<>();
-  public int currentColor;
-  public int currentFaction;
+  public static class CODeets
+  {
+    int co, color, faction;
+  }
+  public ArrayList<CODeets> coList = new ArrayList<>();
```

Comment:
Did we just not support saving and loading of multi-CO teams before this?

Reply:
Nah, I just aggregated the CO data into a class.
What was supported was "Each player has N COs, 1 color, and 1 faction", but now it's N of each.